modules/redisearch.ts

```diff
@@ -708,50 +714,59 @@ export type FTSearchParameters = {
  * @param reduce.nargs The number of arguments of the 'REDUCE' parameter
  * @param reduce.arg The argument of the 'REDUCE' parameter
  * @param reduce.as The name of the function of the 'REDUCE' parameter
- * @param sortby The 'SORTBY' parameter. 
+ * @param sortby The 'SORTBY' parameter.
  * @param sortby.nargs The number of arguments of the 'SORTBY' parameter
  * @param sortby.property The property name of the 'SORTBY' parameter
  * @param sortby.sort The sort type of the 'SORTBY' parameter
  * @param sortby.max The max of the 'SORTBY' parameter
- * @param apply The 'APPLY' parameter. 
+ * @param apply The 'APPLY' parameter.
  * @param apply.expression The expression of the 'APPLY' parameter
  * @param apply.as The as of the 'APPLY' parameter
  * @param limit The 'LIMIT' parameter.
  * @param limit.offset The offset of the 'LIMIT' parameter
  * @param limit.numberOfResults The number of results of the 'LIMIT' parameter
  * @param filter The expression of the 'FILTER' parameter.
  */
-export type FTAggregateParameters = {
+export declare type FTAggregateParameters = {
     load?: {
-        nargs: string,
-        property: string
-    },
+        nargs: string;
+        property: string;
+    };
     groupby?: {
-        nargs: string,
-        property: string
-    },
+        nargs: string;
+        property: string;
+    };
     reduce?: {
-        function: string,
-        nargs: string,
-        arg: string,
-        as: string
-    },
+        function: string;
+        nargs: string;
+        arg: string;
+        as: string;
+    };
     sortby?: {
-        nargs: string,
-        property: string,
-        sort: 'ASC' | 'DESC',
-        max: number
-    },
+        nargs: string;
+        properties: SortByProperty[]
+        max: number;
+    };
     apply?: {
-        expression: string,
-        as: string
-    },
+        expressions: Expressions[];
+    };
     limit?: {
-        offset: string,
-        numberOfResults: number
-    },
-    filter?: string
-}
+        offset: string;
+        numberOfResults: number;
+    };
+    filter?: string;
+};
+
+
+declare type SortByProperty = {
```

Comment:
Instead of declare, write export. Also, don't forget to add it to the index.ts file